app/code/Aspire/Module/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="aspire" translate="label" sortOrder="10">
+            <label>Aspire Module</label>
+        </tab>
+        <section id="module" translate="label" sortOrder="130" showInDefault="1" showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Aspire Admin Configuration</label>
+            <tab>aspire</tab>
+            <resource>Aspire_Module::config_extension</resource>
+            <group id="configuration" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                <label>General Configuration</label>
+                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Module Enable</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
+                <field id="api_url" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="2">
```

Comment:
@kartikaspire showInStore - What are the values accepted by showInStore?. is that support 2? Please explain here

Reply:
The values are accepted only 1 and 0. I have updated to 1.